x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.test.ts

```diff
@@ -850,46 +850,16 @@ describe('Detections Rules API', () => {
 
     test('requests snooze settings of multiple rules by their IDs', () => {
       fetchRulesSnoozeSettings({ ids: ['id1', 'id2'] });
-
-      expect(fetchMock).toHaveBeenCalledWith(
-        expect.any(String),
-        expect.objectContaining({
-          query: expect.objectContaining({
-            filter: 'alert.id:"alert:id1" or alert.id:"alert:id2"',
-          }),
-        })
-      );
-    });
-
-    test('requests the same number of rules as the number of ids provided', () => {
```

Comment:
I combined the three tests into one.

Reply:
@cnasikas Let's please revert changes to this file:

- It doesn't seem related to / in the scope of this PR.
- The new test uses snapshot testing (`toMatchInlineSnapshot`) that is considered an anti-pattern in our team.

Reply:
You are right. I added these tests as part of calling the internal Find Rules API before this PR https://github.com/elastic/kibana/pull/193757. Done in [`b8dc926` (#183756)](https://github.com/elastic/kibana/pull/183756/commits/b8dc9267d85484d5626f3110c04bdf35f3770479). Curious though, why do you consider the `toMatchInlineSnapshot` an anti-pattern in this scenario?